core/src/serve_repair.rs

```diff
@@ -1214,8 +1217,6 @@ impl ServeRepair {
             from_addr,
             nonce,
         )?;
-
-        inc_new_counter_debug!("serve_repair-window-request-ledger", 1);
```

Comment:
Did you mean to remove this?

Reply:
Yes, it would be tracking the inverse of the new metric.